src/libraries/Microsoft.Extensions.Configuration/tests/FunctionalTests/ArrayTests.cs

```diff
@@ -47,7 +47,6 @@ public class ArrayTests : IDisposable
 ";
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/60583", TestPlatforms.iOS | TestPlatforms.tvOS)]
         public void DifferentConfigSources_Merged_KeysAreSorted()
```

Comment:
This one needs to be disabled again based off of the comment that it still fails https://github.com/dotnet/runtime/issues/60583#issuecomment-1206831713